packages/turf-polygonize/index.js

```diff
@@ -0,0 +1,21 @@
+var polygonize = require('polygonize');
+
+/**
+ * Polygonizes {@link LineString|(Multi)LineString(s)} into {@link Polygons}.
+ *
+ * Implementation of GEOSPolygonize function (`geos::operation::polygonize::Polygonizer`).
+ *
+ * Polygonizes a set of lines that represents edges in a planar graph. Edges must be correctly
+ * noded, i.e., they must only meet at their endpoints.
+ *
+ * The implementation correctly handles:
+ *
+ * - Dangles: edges which have one or both ends which are not incident on another edge endpoint.
+ * - Cut Edges (bridges): edges that are connected at both ends but which do not form part of a polygon.
+ *
+ * @name polygonize
+ * @param {FeatureCollection|Geometry|Feature<LineString|MultiLineString>} geojson Lines in order to polygonize
+ * @returns {FeatureCollection<Polygon>} Polygons created
+ * @throws {Error} if geoJson is invalid.
```

Comment:
If you are going to include this `@throws {Error}` in the JSDocs at least include the tests in TurfJS to make those errors fail.

Noticed you're making Polygon fail, any reason why this would fail? I can already envision this error being very annoying, example if you have a collection of LineStrings and you `polygonize` them you then get a FeatureCollection of Polygons, if you keep adding more LineStrings to that FeatureCollection and re-run `polygonize` then it would fail since the FeatureCollection will have mixed contents.

It seems like throwing an Error against Polygons would make less practical.